tests/simulator_test.cpp

```diff
@@ -90,7 +92,7 @@ TEST_F(EnergyCollectorTest, ThrowExceptionWhenInvalidNumCollector) {
 TEST_F(EnergyCollectorTest, NotEvenNumOfEnergyCollectorTest) {
   // This value will be used in actual simulation
   const int numCollector = 37;
-  energyCollectors = buildCollectors(nonEmptyModel, numCollector);
+  auto energyCollectors = buildCollectors(nonEmptyModel, numCollector);
 
   // TODO: THis should be done inside class fixture
```

Comment:
Again, wrong assumptions.

These numbers are as such **only** because you use emptyModel. When you pass some real values, they will change.